src/Workspaces/Core/Portable/Execution/AbstractOptionsSerializationService.cs

```diff
@@ -95,6 +97,7 @@ protected void WriteCompilationOptionsTo(CompilationOptions options, ObjectWrite
             out bool concurrentBuild,
             out bool deterministic,
             out bool publicSign,
+            out MetadataImportOptions metadataImportOptions,
```

Comment:
was there a reason to add this above the existing values?

Reply:
other values were using default and serialized/deserialized.